glutil/text/cli_text.py

```diff
@@ -25,7 +25,6 @@
 and in most cases you should just do that instead of running this script.
 """
 
-
```

Comment:
I think we should leave these extra lines. Leaving them doesn't break pep8 formatting, and it makes it easier to see where each string ends and the next begins.

Reply:
That's fine. The line spacing was previously inconsistent between blocks which is what I was fixing.